.circleci/config.yml

```diff
@@ -510,11 +510,18 @@ jobs:
             -   run:
                     name: Publish dashbords docs
                     command: yarn run deploy-docs-ci
+            -   store_artifacts:
+                    path: ./nova-docs.zip
+    nova-docs-cleanup:
+        working_directory: ~/nova
+        docker:
+            -   image: circleci/node:12
+        environment:
+            NOVA_DIR: ~/nova
+        steps:
             -   run:
                     name: Docs Cleanup
                     command: yarn run clean-docs-ci
-            -   store_artifacts:
-                    path: ./nova-docs.zip
```

Comment:
do we need to store the cleaned up version of the docs in the artifacts?

Reply:
Github review made this look bad